R/subsetting.R

```diff
@@ -199,7 +199,7 @@ NULL
   }
 
   # From here on, i, j and drop contain correct values:
-  xo <- tbl_subset_col(x, j = j)
+  xo <- tbl_subset_col(x, j = j, arg = substitute(x))
```

Comment:
Just curious, does this work with method dispatch?

Reply:
This is passed to `as_label()` upstream, so the arg is abbreviated if it is a large expression. We could change the labelling strategy later if needed.